### PR TITLE
Simplify the wording for locator search

### PR DESCRIPTION
**Pull request by m-kuhn**
"Trigger without its prefix" tends to be quite obscure to casual users.
I couldn't test (cannot build currently).

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for da2da751b60355e0a1c05f4b26bdf620c2944fb9: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-arm-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-arm64-android.apk)
- [all access arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-arm-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4218-6083bff28a40f3fa9f7a2617da148f521ae4373b-x86-android.apk)
</details>

**Comment by nirvn** [comment]
This is how it looks:
image

Trying to think if we can come up with a better 'prefix' tip below. Maybe: 'By typing the XX prefix in the search bar, this locator can still be triggered if not enabled by default' ?

We could also consider showing the tip only when the enabled by default checkbox is turned off.

Finally, I would move the description to sit _above_ the checkbox, IMHO it would tie things together much more nicely.

**Comment by m-kuhn** [comment]
@jeroen-groenebij, as discussed

**Comment by nirvn** [comment]
@m-kuhn , updated looks:
image

**Comment by nirvn** [comment]
@m-kuhn , looking at this, I'm thinking the disabled tip label would probably fit better above the checkbox too, it's otherwise a bit challenging to see what belongs to which locator, sorry.

As for the improved tip, I think we need to shuffle the even around a bit, it should be: 'By typing the XX prefix in the search bar, this locator filter can be used even when disabled'.

**Comment by Jeroen-GroeneBij** [comment]
The disabled tip label could be:
When disabled, this locator filter can still be used by typing the prefix XX in the search bar.

**Review by nirvn — APPROVED** [review]

**Comment by nirvn** [comment]
I'll merge now so we can give as much time as possible for the community to translate these tweaked strings prior to QField 2.8.
